### PR TITLE
Ncl 929

### PR DESCRIPTION
**Pull request by pgier**

### CODE REVIEW

File: test-arquillian-container/pom.xml [hunk 1]
```diff
@@ -198,6 +205,38 @@
                 </configuration>
               </execution>
               <execution>
+                <id>deploy-hsql-jdbc-driver</id>
+                <phase>generate-test-resources</phase>
+                <goals>
+                  <goal>deploy-artifact</goal>
+                </goals>
+                <configuration>
+                  <groupId>org.hsqldb</groupId>
```
Comment (slaskawi) [inline test-arquillian-container/pom.xml]: Why do we need it? Can't we use default Wildfly's DS?

File: pom.xml [hunk 2]
```diff
@@ -658,6 +658,13 @@
         <version>${version.org.json}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.hsqldb</groupId>
```
Comment (slaskawi) [inline pom.xml]: The same question - what is the advantage of using explicit Database to Wildfly/EAP's default one?

File: test-arquillian-container/pom.xml (hunk 1 above)
Comment (pgier) [inline test-arquillian-container/pom.xml]: H2 does not support the data type "timestamp with time zone".

**Comment by alexcreasy** [comment]
This PR causes failures for me when I try to deploy locally and also prevents from deploying on the autodeploy server, I'm reverting this PR for now.

**Pull request merged**